### PR TITLE
MRG: Add function for plotting paths

### PR DESCRIPTION
**Pull request by togawa28**
Added the function to visualize paths in the original form. For details see #170.
@jJasonWang please review.

### CODE REVIEW

File: mousestyles/visualization/plot_path.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    xlim = [-16.24, 3.76], ylim = [0.9, 43.5]):
+    r"""
+    Plot the lines along paths.
+
+    Parameters
+    ----------
+    movement : pandas.DataFrame
+        CX, CY coordinates. Must have length greater than 1.
+
+    sep_points : list
```
Comment (jJasonWang) [inline mousestyles/visualization/plot_path.py]: `sep_points` is the list of path indices from the function `path_index`, right? Maybe we can change the description to make it consistent. 
The following description is what I put in the `filter_path` function.

> paths: list
        a list containing the indices for all paths

**Comment by togawa28** [comment]
I've fixed styles by following advises of @jJasonWang. The coverage decreased because I didn't put any test functions: this is a function to make a plot and is difficult to be tested.
Do you guys think we need some additional functionalities as discussed the issue #170? If you all think we've done with this function and no more additional stuff, I'll change it to MRG.

File: mousestyles/visualization/plot_path.py (hunk 1 above)
Comment (togawa28) [inline mousestyles/visualization/plot_path.py]: Modified. See the newest one.

**Comment by togawa28** [comment]
This is ready to merge. Discussions can be found in #170.

**Comment by jarrodmillman** [comment]
@togawa28 Could you add a plot to the report to show that this works?  The instruction are in the `README.md`:

```
$ cat doc/source/report/README.md 
Notes
=====


For generating plots, please add a script that generates the desired plot
in `report/plots/example.py`.

Then add a directive like this in the file where you would like the figure
to appear:

``
.. plot:: report/plots/example.py

   Put your caption here.
``
```

Once you've done that (it should be trivial to add), rebase and change this back to `MRG`.

**Comment by togawa28** [comment]
@jarrodmillman I added .py file in plots folder, modified path.rst file, and rebased it. I'm not sure why, but my .rst file does not show my plot, although .py file looks working fine. Except for that, it's ready to merge so the title is changed to MRG.

**Pull request merged**